### PR TITLE
Revert "[펭귄] Test pull request reviewer"

### PR DESCRIPTION
**Pull request by CoodingPenguin**
Reverts boostcamp-ai-tech-4/ai-tech-interview#9

### CODE REVIEW

**Review by bsm8734 — APPROVED** [review]
good.

File: answers/statistics-math.md [hunk 1]
```diff
@@ -55,8 +55,6 @@
 
 #### 누적 분포 함수와 확률 밀도 함수는 무엇일까요? 수식과 함께 표현해주세요
 
-Pull request test
-
```
Comment (bsm8734) [inline answers/statistics-math.md:59]: 리뷰리뷰

**Review by CoodingPenguin — COMMENTED** [review]
오 좋아용

**Review by bsm8734 — APPROVED** [review]

File: answers/statistics-math.md (hunk 1 above)
Comment (CoodingPenguin) [reply answers/statistics-math.md:59]: 리뷰리뷰2

Comment (CoodingPenguin) [reply answers/statistics-math.md:59]: resolve conversation 실험

**Comment by Heeseok-Jeong** [comment]
Good

**Comment by CoodingPenguin** [comment]
test

**Comment by CoodingPenguin** [comment]
test

**Pull request merged**